src/System.Data.SqlClient/src/System/Data/SqlClient/SqlFileStream.Unix.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file looks identical to the .Uap.cs one.  You could have a single SqlFileStream.Unsupported.cs file that's included in both the UAP and Unix builds.

That said, why can't it be supported?  There's no way to make the majority of the functionality work on Unix or in UAP?

Reply:
I will create a singe file `SqlFileStream.Unsupported.cs`
As to why can't it be supported answered in the comments below.